pegasus/sites.v3/code.org/public/translate/supported.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Language support for each tutorial
+nav: translate_nav
+theme: responsive
+style_min: true
+---
+
+# Language support for each tutorial
+
+<%= view :csf_languages %>
+
+<%= view :frozen_languages %>
+
+<%= view :starwars_languages %>
+
+<%= view :minecraft_adventurer_languages %>
+
+<%= view :minecraft_designer_languages %>
```

Comment:
Place in this order:

1) CSF
2) Minecraft Designer
3) Minecraft Adventurer
4) Star Wars
5) Frozen